app/routes.py

```diff
@@ -1,2 +1,58 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request, abort
+from app.routes_helpers import validate_model
 
+planets_bp = Blueprint('planets', __name__, url_prefix='/planets')
+
+@planets_bp.route("", methods=["GET"])
+def read_all_planets():
+    name_query = request.args.get("name")
+
+    if name_query:
+        planets = Planet.query.filter_by(name=name_query)
+    else:
+        planets = Planet.query.all()
+
+    planets_response = [planet.to_dict() for planet in planets]
+
+    return jsonify(planets_response)
+
+@planets_bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+    new_planet = Planet.from_dict(request_body)
+
+    db.session.add(new_planet)
+    db.session.commit()
+
+    return make_response(f"Planet {new_planet.name} successfully created", 201)
```

Comment:
👀 Note that this is still returning a bare HTML result here. At the least, we should consistently return JSON from all our endpoints (use `jsonify`, or pack the message into aq `dict` like we do in the validation helper).

But here, we might also decide to return the created planet record, as this would give the caller access to the `id` that was created, as well as any default values they may not have included in their request.